NEWS.md

```diff
@@ -1,5 +1,9 @@
 # testthat (development version)
 
+* New `test_local()` tests a local source package. It's equivalent to
```

Comment:
Maybe clarify that it is not a package file, but a directory? Not terribly important in the NEWS, though.

Reply:
:+1: